pinc/3rdparty/iconify/octicon-details.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file has to be named `details.json` for it to show up in the Credits page. I know this license only applies to the `octicon` ones for now, but let's punt that down the road and just rename the file for now.

Reply:
Done this. I should have read the documentation.